Sources/AppStoreConnectCLI/Model/BuildDetailsInfo.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  BuildDetails.swift
+//  AppStoreConnect-Swift-SDK
+//
+//  Created by Nafisa Rahman on 27/4/20.
+//
+
+import AppStoreConnect_Swift_SDK
+import Foundation
+import SwiftyTextTable
+
+struct BuildDetailsInfo: ResultRenderable {
+  let build: Build?
+  let app: AppStoreConnect_Swift_SDK.App?
+  let prereleaseVersion: AppStoreConnect_Swift_SDK.PrereleaseVersion?
+  let buildBetaDetail: AppStoreConnect_Swift_SDK.BuildBetaDetail?
+  let betaAppReviewSubmission: AppStoreConnect_Swift_SDK.BetaAppReviewSubmission?
```

Comment:
For our models we should try to list out all the things we are interested in as a flat properties. We can nest into our own types as they make sense.

Have a look at our `App` model and our `BetaTester` model